config/initializers/rails_config.rb

```diff
@@ -1,5 +1,5 @@
 require 'rails_config'
 
 RailsConfig.setup do |config|
-  config.const_name = "Settings"
+  config.const_name = 'Settings'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.